Dockerfile

```diff
@@ -12,19 +12,27 @@ ENV NTP_DIRECTIVES="ratelimit\nrtcsync"
 
 # install chrony
 RUN apk add --no-cache chrony tzdata && \
-    rm /etc/chrony/chrony.conf
+    rm /etc/chrony/chrony.conf && \
+    chmod 1750 /etc/chrony && \
```

Comment:
If chowned, we can probably forego needing to set the sticky bit. I'll give that a try.